KGP.TicketAPP.Utils/Validation/ValidationService.cs

```diff
@@ -6,32 +6,20 @@
 
 namespace KGP.TicketAPP.Utils.Validation
 {
-    public class ValidationService : IValidationService
+    public static class ValidationService
```

Comment:
czy to musi być statyczne? nie zadziała, jak zrobisz dependency injection tam, gdzie tego używasz?

Reply:
moge jutro sprobowac, tylko nie wiem czy dla tego actionfiltru moge to wstrzyknac. jak sie nie bedzie dalo to imo lepiej statyczne, zeby dla kazdego wywolania nie tworzyc jednej instancji